src/main/scala/NIC.scala

```diff
@@ -448,12 +491,12 @@ trait CanHavePeripheryIceNICModuleImp extends LazyModuleImp {
 
     if (nicConf.usePauser) {
       val pauser = Module(new PauserComplex(qDepth))
-      pauser.io.ext.flipConnect(NetDelay(netio, latency))
+      pauser.io.ext.flipConnect(NetDelay(NICIO(netio), latency))
       pauser.io.int.out <> pauser.io.int.in
       pauser.io.macAddr := netio.macAddr + (1 << 40).U
       pauser.io.settings := netio.pauser
     } else {
-      netio.in <> Queue(LatencyPipe(netio.out, latency), qDepth)
+      netio.in <> Queue(LatencyPipe(NICIO(netio).out, latency), qDepth)
     }
     netio.in.bits.keep := NET_FULL_KEEP
   }
```

Comment:
You need to fix up the NICIO signal in the connectSimNetwork method below as well.

Reply:
SimNetwork was broken in other ways, as well